themes/default/content/docs/intro/console/organizations.md

```diff
@@ -9,92 +9,112 @@ aliases:
-to your organization's stacks
+## Organization Roles
+
+| Role | Description |
+|--------|--------|
+| Admin | Administrators have full access to the organization including: [Inviting members]({{< relref "/docs/intro/console/organization-roles#organization-membership" >}}), [creating teams](({{< relref "/docs/intro/console/teams#creating-a-team" >}})) and policies, managing stack permissions and [role-based access control](({{< relref "/docs/intro/console/organization-roles" >}})), adjusting billing information and controlling the organization settings. |
+| Member | Members are able to view and edit stacks they have access to and view members and teams. |
 
 ## Creating a New Organization
 
-You can create a new Pulumi organization directly from the Pulumi Console.
+You can create a new Pulumi organization directly from the Pulumi Console by opening the organization menu in the top-navigation and choosing **New organization...*
+
+## Moving Between Organizations
+
+The Organization drop-down list displays all of the organizations your account is
+associated with, and lets you add a new organization backed by a third
+party identity provider. To switch to a different organization:
+
+1. Select the organization drop-down list in the upper left corner of the Console screen
+next to the Pulumi logo.
+1. Select your organization name.
+
+## Getting Access to an Organization
+
+To become a member of a Pulumi organization, you must be invited by an existing Pulumi
+organization administrator, or you must submit a request to the administrator for approval.
+In addition, depending you may also need to be a member of the third-party organization or group backing the Pulumi organization.
 
-<a class="btn btn-secondary" href="https://app.pulumi.com/site/trial"
-target="_blank">
-    Create organziation
-</a>
+For example, to become a member of a Pulumi organization backed by a GitLab Group,
+you must associate a GitLab identity with your Pulumi account, and also
+be a member of that GitLab group.
 
-## Organization Types
+### Stack Permissions
 
-A Pulumi organization needs to be linked to a third-party identity provider, offering an
+Any organization member with the `Admin` role automatically has `Admin`
+permissions for all of the organization's stacks. Regular organization members
+are granted the organization's _base permissions_ instead.
+
+For example, if the organization's base permissions is `Write`, then
+any organization member can update any organization stack.
+
+If the stack permission for all members is `None`, then organization members must be
+granted access using [teams]({{< relref "teams" >}}) in order to update, or even [view the organization
+stacks]({{< relref "project-and-stack-management#viewing-your-organization-stacks" >}}).
+
+Additionally, organization admins can toggle whether organization members can create stacks, whether
+stack admins can delete stacks, and whether stack admins can move stacks to other organizations.
```

Comment:
i think i can shorten this and move it to the roles section; since we also have the stack permissions page